docs/farming-&-staking/farming/space-acres/space-acres-install.mdx

```diff
@@ -33,54 +34,69 @@ What to expect:
 - Sync might take from a few hours to up to a day.
 - The initial syncing and plotting process requires substantial computational resources, so you may notice that your computer runs slower for day-to-day tasks until syncing and plotting are complete.
 
-
 :::note
 Please note that a wallet with a Substrate address is required to run the software.<br />
 To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](/docs/farming-&-staking/wallets/subwallet.md) or [**`{polkadot.js}`** extension](/docs/farming-&-staking/wallets/polkadot.md).
 :::
 
 
+### Installation Videos
+
+For those who prefer a visual guide, we have prepared installation videos to assist you. The videos will help you to set up Space Acres. Please select your operating system below to watch the corresponding installation video. If you prefer detailed text instructions, please proceed to [Step 1](#step-1-download-and-install-space-acres).
```

Comment:
Did you have the install video in the OS specific tabs originally? We only have one so far.

Reply:
Sorry, yep they've been in tabs originally, but I though it was confusing. Adjusted the section 👍